www_src/pages/section-2/section-2.jsx

```diff
@@ -295,6 +314,13 @@ var App = React.createClass({
   changeZoom: function (event) {
     this.refs.masterGrid.zoom(event.amount);
   },
+  showOverview: function () {
+    this.refs.masterGrid.setState({
```

Comment:
it's better to make this a real function instead of calling setState on "not yourself", so calling `this.refs.masterGrid.setValues(1, 0, 0)` and then in Grid having a `this.setValues: function(zoom, cameraX, cameraY) { this.setState(...); }`